### DISCUSSION TITLE
Decide whether to open link in current window or in default browser based on link

### DISCUSSION THREAD

**Discussion by Nystrata** [filips123/PWAsForFirefox#526] [Ideas]
A thing I notice with PWA is that

If I have a PWA for `https://www.twitch.tv/` and I click on a link that takes me withing twitch.tv, like `https://www.twitch.tv/following`, then it behaves as usual.
If I have a PWA for `https://www.twitch.tv/` and I click on a link that takes me *outside* of twitch.tv, like `https://www.google.com/`, then it behaves as usual but displays an additional address bar to indicate that I'm outside the usual PWA address

My suggestion is to have an configurable option, so that
When I click on a link that takes me out of the usual PWA addresss (like clicking on a google.com link in my twitch.tv PWA), 
instead of opening the outgoing link in the current window,
it'll open that in my default browser instead of opening in the current window.

**Comment by filips123** [comment]
You can enable [Open out-of-scope URLs in a default browser](https://pwasforfirefox.filips.si/user-guide/browser/#open-out-of-scope-urls-in-a-default-browser) in the app browser settings.

**Reply by ke7in-zz** [reply]
Does this have to be configured for each PWA? If so, for a PWA with no URL bar, how do we get to about:config?

**Reply by filips123** [reply]
It needs to be configured for each PWA profile. You can press `F6` and then enter `about:config` in the dialog.

**Reply by ke7in-zz** [reply]
Thanks! I think this should be the default behavior, or expose a checkbox in the extension menu?

**Reply by ke7in-zz** [reply]
> It needs to be configured for each PWA profile. You can press `F6` and then enter `about:config` in the dialog.

Hmm this is not solving my issue. I have a webapp for TradingView which has integration with a broker where the login spawns a pop-up. Since the login happens in the default browser, the authentication doesn't work. 

Is there a way to just open links in a new window for the same profile?

**Reply by filips123** [reply]
You need to check which domain that login page is on and add it to the "Domains always allowed to be opened in the app browser" option in the app browser settings. You should add just domain, without any `https://` prefix or path.
